site/assignments/zombies/zombies.rst

```diff
@@ -1,33 +1,197 @@
+    For this assignment, you will need to use special Python packages (networkx, matplotlib). The good news is that if
+    you are using Colab you do not need to do anything special. If, however, you are using a local Python interpreter,
+    you may need to install the packages yourself.
+
+
+Task
+====
+
+
+Provided Files
+==============
+
+
+Part 0 --- Read the Assignment
+==============================
+
+Read the assignment description in its entirety before starting.
+
+
+Part 1 --- Uploading Files to Colab
+===================================
+
+
+Part 2 --- Read Over Provided Code
+==================================
+
+
+Part 3 --- Check if a City is Infected
+======================================
+
+
+Part 4 --- Get a City's Neighbours
+==================================
+
+
+Part 5 --- Infecting and Curing a City
+======================================
+
+
+Part 6 --- How Many Cities are Infected
+=======================================
+
+* Both ``number_of_cities_infected`` and ``is_world_completely_infected`` here
+
+
+Part 7 --- Simulation Step
+==========================
+
+
+Part 8 --- Simulate Infections Disease Scenario
+===============================================
+
+
+Part 9 --- Using Your Code and Plotting
+=======================================
+
+
+Part 10 --- Testing
+===================
+
+
+Some Hints
+==========
+
+* Work on one method at a time
+* Get each method working perfectly before you go on to the next one
+* Test each method as you write it
+    * This is a really nice thing about programming; you can call your methods and see what result gets returned
+    * Mentally test before you even write --- what does this method do? What problem is it solving?
+
+* If you need help, ask
+    * Drop by office hours
+
+
+Some Marking Details
+====================
 
 .. warning::
+    Just because your program produces the correct output, that does not necessarily mean that you will get perfect, or
+    even that your program is correct.
+
+Below is a list of both *quantitative* and *qualitative* things we will look for:
+
+* Correctness?
+* Did you follow instructions?
+* Comments?
+* Variable Names?
+* Style?
+* Did you do just weird things that make no sense?
+
+
+What to Submit to Moodle
+========================
+
+* Make sure your **NAME** and **STUDENT NUMBER** appear in a comment at the top of the program
+* Submit your version of ``asn2.py`` to Moodle
+    * Do **not** submit the .ipynb file
+    * To get the ``asn2.py`` file from Colab, see the example image in Assignment 1
+
+* A text/word file answering the analysis questions and providing *data* to back up your answers
+    * Include images in your submission
+
+
+.. warning::
+
+    Verify that your submission to Moodle worked. If you submit incorrectly, you will get a 0.
+
+
+General FAQ:
+============
+
+* My drawing looks different every time!
+   * I know, it should, it's random every time.
```

Comment:
Remove "I know."

This whole section needs a touchup, can be follow ups as you mentioned.